### ISSUE TITLE
want a graph of reference relationships between functions

### ISSUE THREAD

**Issue by doki23** [intellij-rust/intellij-rust#8632]
I really want a graph which can describe the reference relationships between functions.
For example, there 3 functions, f1, f2, f3, f1 calls f2, f2 calls f2, f3, it looks like
f1->( f2 <=>  f2 )->f3

**Comment by Kobzol** [comment]
Something like callee/caller hierarchy? https://github.com/intellij-rust/intellij-rust/pull/7442

**Comment by doki23** [comment]
> Something like callee/caller hierarchy? https://github.com/intellij-rust/intellij-rust/pull/7442

yes. i found an idea plugin 'Call Graph', but it only supports Java.
